src/test/java/newbank/server/TestNewBank.java

```diff
@@ -24,33 +24,33 @@ public static void setupBank() throws DuplicateCustomerException, PasswordInvali
   public void canRegisterNewCustomer()
       throws DuplicateCustomerException, PasswordInvalidException, UsernameInvalidException {
     assertThat(bank.getCustomer("customer2").isPresent(), equalTo(false));
-    bank.addCustomer("customer2", "123456");
+    bank.addCustomer("customer2", "Abc123");
     assertThat(bank.getCustomer("customer2").isPresent(), equalTo(true));
   }
 
   @Test(expected = DuplicateCustomerException.class)
   public void throwsWhenAddingDuplicateCustomer()
       throws DuplicateCustomerException, PasswordInvalidException, UsernameInvalidException {
-    bank.addCustomer("customer3", "123456");
-    bank.addCustomer("customer3", "123456");
+    bank.addCustomer("customer3", "Abc123");
+    bank.addCustomer("customer3", "Abc123");
   }
 
   @Test(expected = UsernameInvalidException.class)
   public void throwsWhenRegisteringWithInvalidName()
       throws DuplicateCustomerException, PasswordInvalidException, UsernameInvalidException {
-    bank.addCustomer("12345", "123456");
+    bank.addCustomer("12345", "Abc123");
   }
 
   @Test(expected = PasswordInvalidException.class)
   public void throwsWhenRegisteringWithInvalidPassword() throws DuplicateCustomerException, PasswordInvalidException, UsernameInvalidException {
-	  bank.addCustomer("customer4", "");
```

Comment:
The password is deliberately wrong here.

Kindest Regards,
Jules

Reply:
![image](https://user-images.githubusercontent.com/80009035/114574610-4d79d680-9c71-11eb-99fb-8000c22786d3.png)
![image](https://user-images.githubusercontent.com/80009035/114574821-7e5a0b80-9c71-11eb-8cc7-e3d6d8a5ec4f.png)

Thanks for highlighting now updated in my latest push !

Reply:
@claudist The integration tests are now passing but the unit-tests tests are still broken which is why it's failing on here.
The unit-tests can be found here:
![image](https://user-images.githubusercontent.com/58267482/114576836-68e5e100-9c73-11eb-9e6a-da3521e3b3e2.png)

Regards,
Jules

Reply:
ahhh i see ! Will push fix. Thanks!

Reply:
![image](https://user-images.githubusercontent.com/80009035/114579641-f88c8f00-9c75-11eb-99f0-df9cc2fad4fb.png)
![image](https://user-images.githubusercontent.com/80009035/114579699-05a97e00-9c76-11eb-9840-44b730b0fbbb.png)

Just pushed the latest, should be good now.